Application/EdFi.Ods.Tests/EdFi.Ods.Security/Authorization/EdFiAuthorizationProviderTests.cs

```diff
@@ -468,19 +468,19 @@ public abstract class When_authorizing_a_request_affected_by_authorization_strat
                             {
                                 new ResourceClaimAuthorizationMetadata
                                 {
-                                    ClaimName = Resource1ClaimUri, AuthorizationStrategy = null //"First"
+                                    ClaimName = Resource1ClaimUri, AuthorizationStrategies = null //"First"
                                 },
                                 new ResourceClaimAuthorizationMetadata
                                 {
-                                    ClaimName = Resource2ClaimUri, AuthorizationStrategy = "Second"
+                                    ClaimName = Resource2ClaimUri, AuthorizationStrategies = new [] { "Second" }
                                 },
                                 new ResourceClaimAuthorizationMetadata
                                 {
-                                    ClaimName = Resource3ClaimUri, AuthorizationStrategy = null
+                                    ClaimName = Resource3ClaimUri, AuthorizationStrategies = null
                                 },
                                 new ResourceClaimAuthorizationMetadata
                                 {
-                                    ClaimName = Resource4ClaimUri, AuthorizationStrategy = "Fourth"
+                                    ClaimName = Resource4ClaimUri, AuthorizationStrategies = new List<string> { "Fourth" }
```

Comment:
Convert to an array rather than creating lists.

Reply:
I noticed that you have updated it